runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -225,15 +249,22 @@ def model_predict(self, df: pd.DataFrame, horizon: int = None) -> pd.DataFrame:
         result_df["ts_id"] = str(df["ts_id"].iloc[0])
         return result_df[return_cols]
 
-    def predict(self, context: mlflow.pyfunc.model.PythonModelContext, input_df: pd.DataFrame) -> pd.DataFrame:
+    def predict(self, context: mlflow.pyfunc.model.PythonModelContext, model_input: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Just to make sure. Is the conclusion that we'll return a pd.Series of yhat for the predict API instead of a pd.DataFrame?
In this case, shall we change the type hints of the predict methods? For both MultiSeriesProphetModel and ProphetModel.